activerecord/lib/active_record/connection_adapters/sqlite3_adapter.rb

```diff
@@ -62,7 +62,8 @@ class SQLite3Adapter < AbstractAdapter
         time:         { name: "time" },
         date:         { name: "date" },
         binary:       { name: "blob" },
-        boolean:      { name: "boolean" }
+        boolean:      { name: "boolean" },
+        bigint:       { name: "bigint" },
```

Comment:
Currently, `:bigint` and `:bigserial` entries does not affects the behavior because the type of these native column is an `:integer`.

My hope is that avoiding inconsistency.

Should we remove `:bigint` and `:bigserial` entries?